file/file.h

```diff
@@ -53,8 +53,12 @@ SOFTWARE.
 namespace bricks {
 
 // Platform-indepenent, injection-friendly filesystem wrapper.
-// TODO(dkorolev): Move the above methods under FileSystem.
 struct FileSystem {
+#ifndef BRICKS_WINDOWS
+  static constexpr char PathSeparatingSlash = '/';
+#else
+  static constexpr char PathSeparatingSlash = '\\';
+#endif
   static inline std::string GetFileExtension(const std::string& file_name) {
     const size_t i = file_name.find_last_of("/\\.");
```

Comment:
I thought of it and prefer to keep it this way. There's a nonzero chance someone might use `GetFileExtension` on a URL, not to mention that using various slashes as non-slashes as part of the file name (and extension!) is something I'd be glad to relentlessly punish a developer for.